CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ BREAKING CHANGES
   - Adds a new container named `consul-ecs-health-sync` to the task definition which will be responsible for syncing back ECS container health checks into Consul. This container will wait for a successful exit of `consul-ecs-mesh-init` container before starting.
 
 FEATURES
+* Add support for transparent proxy in ECS tasks based on EC2 launch types. Following are the changes made to the `mesh-task` submodule [[GH-264](https://github.com/hashicorp/terraform-aws-consul-ecs/pull/264)]
+  - Adds the following variables [[GH-209](https://github.com/hashicorp/terraform-aws-consul-ecs/pull/209)]
+    - `enable_transparent_proxy` - Defaults to `true`. Fargate based tasks should explicitly pass `false` to avoid validation errors during terraform planning phase.
```

Comment:
@Ganeshrockz should this be a breaking change? Since it will change all the routing, e.g. everything will be routed to/from the proxy.

Reply:
That makes sense. Will move this under the Breaking changes heading in the next PR